src/libraries/System.Net.Http/src/System/Net/Http/BrowserHttpHandler/HttpClientHandler.Browser.cs

```diff
@@ -0,0 +1,168 @@
+        {
+            get => throw new PlatformNotSupportedException();
+            set { throw new PlatformNotSupportedException(); }
+        }
+
+        [UnsupportedOSPlatform("browser")]
+        public ICredentials? Credentials
+        {
+            get => throw new PlatformNotSupportedException();
+            set { throw new PlatformNotSupportedException(); }
+        }
+
+        [UnsupportedOSPlatform("browser")]
+        public int MaxAutomaticRedirections
+        {
+            get => throw new PlatformNotSupportedException();
+            set { throw new PlatformNotSupportedException(); }
+        }
+
+        [UnsupportedOSPlatform("browser")]
+        public int MaxConnectionsPerServer
+        {
+            get => throw new PlatformNotSupportedException();
+            set { throw new PlatformNotSupportedException(); }
+        }
+
+        [UnsupportedOSPlatform("browser")]
+        public int MaxResponseHeadersLength
+        {
+            get => throw new PlatformNotSupportedException();
+            set { throw new PlatformNotSupportedException(); }
+        }
+
+        public ClientCertificateOption ClientCertificateOptions
+        {
+            get => _clientCertificateOptions;
+            set
+            {
+                switch (value)
+                {
+                    case ClientCertificateOption.Manual:
+                        _clientCertificateOptions = value;
+                        break;
+
+                    case ClientCertificateOption.Automatic:
+                        _clientCertificateOptions = value;
+                        break;
+
+                    default:
+                        throw new ArgumentOutOfRangeException(nameof(value));
+                }
+            }
+        }
+
+        [UnsupportedOSPlatform("browser")]
+        public X509CertificateCollection ClientCertificates => throw new PlatformNotSupportedException();
+
+        [UnsupportedOSPlatform("browser")]
+        public Func<HttpRequestMessage, X509Certificate2?, X509Chain?, SslPolicyErrors, bool>? ServerCertificateCustomValidationCallback
+        {
+            get => throw new PlatformNotSupportedException();
+            set { throw new PlatformNotSupportedException(); }
+        }
+
+        [UnsupportedOSPlatform("browser")]
+        public bool CheckCertificateRevocationList
+        {
+            get => throw new PlatformNotSupportedException();
+            set { throw new PlatformNotSupportedException(); }
+        }
+
+        [UnsupportedOSPlatform("browser")]
+        public SslProtocols SslProtocols
+        {
+            get => throw new PlatformNotSupportedException();
+            set { throw new PlatformNotSupportedException(); }
+        }
+
+        [UnsupportedOSPlatform("browser")]
+        protected internal override HttpResponseMessage Send(HttpRequestMessage request, CancellationToken cancellationToken)
+            => throw new PlatformNotSupportedException();
```

Comment:
NIT: Here are `=>` on the new line; in `HttpClient.Browser.cs` on the previous one. We should choose one formatting and stick to it.